python/tvm/autotvm/measure/measure.py

```diff
@@ -56,7 +56,7 @@ class MeasureResult(namedtuple("MeasureResult", ["costs", "error_no", "all_cost"
     def __repr__(self):
         error_no_str = (
             str(self.error_no)
-            if self.error_no not in MeasureErrorNo
+            if not isinstance(self.error_no, (MeasureErrorNo, int))
```

Comment:
Can you rewrite this without a `not` in your check?

Also write a comment that explains what happens if it's neither an int or a MeasureErrorNo? (Is it just an exception?)